src/main/java/com/nutomic/syncthingandroid/syncthing/AppConfigReceiver.java

```diff
@@ -0,0 +1,101 @@
+                        break;
+
+                    case "stop":
+                        if (SyncthingService.alwaysRunInBackground(context)) {
+                            final String msg = context.getString(R.string.appconfig_receiver_background_enabled);
+
+                            NotificationCompat.Builder nb = new NotificationCompat.Builder(context)
+                                    .setWhen(System.currentTimeMillis())
+                                    .setContentText(msg)
+                                    .setTicker(msg)
+                                    .setStyle(new NotificationCompat.BigTextStyle().bigText(msg))
+                                    .setContentTitle(context.getText(context.getApplicationInfo().labelRes))
+                                    .setSmallIcon(context.getApplicationInfo().icon)
+                                    .setAutoCancel(true)
+                                    .setContentIntent(PendingIntent.getActivity(context, 0, new Intent(context, MainActivity.class), PendingIntent.FLAG_UPDATE_CURRENT));
+
+                            if (Build.VERSION.SDK_INT >= 21) {
+                                nb.setCategory(Notification.CATEGORY_ERROR);        // Only supported in API 21 or better
+                            }
+
+                            NotificationManager nm = (NotificationManager) context.getSystemService(Context.NOTIFICATION_SERVICE);
+                            nm.notify(ID_NOTIFICATION_BACKGROUND_ACTIVE, nb.build());
+
+                        } else {
+                            Intent intent1 = new Intent(context, SyncthingService.class);
+                            intent1.setAction(SyncthingService.ACTION_SHUTDOWN);
+                            context.startService(intent1);
+                        }
+
+                        break;
+
+                    case "stop2":
+                        if (SyncthingService.alwaysRunInBackground(context)) {
+                            final String msg = context.getString(R.string.appconfig_receiver_background_enabled);
+
+                            NotificationCompat.Builder nb = new NotificationCompat.Builder(context)
+                                    .setWhen(System.currentTimeMillis())
+                                    .setContentText(msg)
+                                    .setTicker(msg)
+                                    .setStyle(new NotificationCompat.BigTextStyle().bigText(msg))
+                                    .setContentTitle(context.getText(context.getApplicationInfo().labelRes))
+                                    .setSmallIcon(context.getApplicationInfo().icon)
+                                    .setAutoCancel(true)
+                                    .setContentIntent(PendingIntent.getActivity(context, 0, new Intent(context, MainActivity.class), PendingIntent.FLAG_UPDATE_CURRENT));
+
+                            if (Build.VERSION.SDK_INT >= 21) {
```

Comment:
Well, even Android Studio itself only gives the numerical value. But agreed, it looks nicer.
Changed